### PR TITLE
Apply rate-limit config to `auth-api` routes, using Contour

### PR DESCRIPTION
**Pull request by dotansimha**
This change is using the Local config for rate-limit: https://projectcontour.io/docs/1.29/config/rate-limiting/#local-rate-limiting

We might want to use the Global config with a proper service in the future. 

This PR also enabled sticky sessions for Contour (requests for the same user session will be routed to the same container), we can safely use the Local option at the moment.

### CODE REVIEW

File: deployment/services/proxy.ts [hunk 1]
```diff
@@ -89,6 +89,10 @@ export function deployProxy({
         service: graphql.service,
         requestTimeout: '60s',
         retriable: true,
+        rateLimit: {
+          maxRequests: 7,
```
Comment (kamilkisiela) [inline deployment/services/proxy.ts:93]: That’s super low, no? The session is validated on every page visit (the initial one) and there’s a chance we get a oauth callback with a redirect to a new page that increases the amount by a lot

**Review by kamilkisiela — COMMENTED** [review]
Looks good, but we should increase the limit. Maybe to 60 requests per minute or something?

File: deployment/services/proxy.ts (hunk 1 above)
Comment (dotansimha) [reply deployment/services/proxy.ts:93]: The calls for `/auth-api` made only when you first load the page (and then it's SPA). It's 3 calls, so you are right, it needs adjustments. 
How about max 10 requests a minute?

Comment (dotansimha) [reply deployment/services/proxy.ts]: @kamilkisiela we can also set `burst: 3` and it will ignore the specific burst config from the token validation

Comment (kamilkisiela) [reply deployment/services/proxy.ts]: Let's test it on staging and see if none of the auth flows are affected

Comment (dotansimha) [reply deployment/services/proxy.ts]: @kamilkisiela seems like `maxRequests: 10` with burst works well (even when I'm refreshing very quickly). Also tried to login/logout multiple times.
When trying to send requests from curl, it blocks me at some point.

**Review by n1ru4l — APPROVED** [review]

**Review by kamilkisiela — APPROVED** [review]

**Pull request merged**